bundles/org.openhab.binding.miio/src/main/resources/database/philips.light.bulb.json

```diff
@@ -105,20 +104,6 @@
 						"parameterType": "NUMBER"
 					}
 				]
-			},
-			{
-				"property": "",
-				"friendlyName": "Toggle",
-				"channel": "toggle",
-				"type": "Switch",
-				"refresh": false,
-				"ChannelGroup": "actions",
-				"actions": [
-					{
-						"command": "toggle",
-						"parameterType": "EMPTY"
-					}
-				]
```

Comment:
Was this moved somewhere else?

Reply:
I checked the latest versions of the apk files and the toggle command cannot be found anymore.
I tested it on my bulb and it did not react, so it seems it is removed from the device

Reply:
I think it was always bit of an odd command... as the (mihome) app could not be sure what was the state after sending it... like if in the mean time the status on the device was already changed e.g. by light switch, the app could think it was switching it off whereas it then would in reality toggle it back on.